src/scripts/validation.js

```diff
@@ -1,13 +1,17 @@
 export const isInputsCorrect = inputs => {
     const emptyInputs = Array.from(inputs).filter(input => input.value === '');
-    if (emptyInputs.length > 0) {
-        const errorParagraph = /* HTML */ `
-            <p class="form__error">The field cannot be empty</p>
-        `;
-        emptyInputs.forEach(emptyInput =>
-            emptyInput.parentNode.insertAdjacentHTML('afterend', errorParagraph)
-        );
-        return false;
+    if (emptyInputs.length === 0) {
+        return true;
     }
-    return true;
+    return false;
+};
+
+export const createErrorParagraphs = inputs => {
+    const emptyInputs = Array.from(inputs).filter(input => input.value === '');
```

Comment:
to tę też wydziel jeszcze do `getEmptyInputs` w tym pliku
ewidentne powtórzenie :)